templates/CRM/Contribute/Form/Task/Invoice.tpl

```diff
@@ -58,7 +58,7 @@
 </table>
 
 <div class="spacer"></div>
-<div class="form-item">
+<div class="crm-submit-buttons">
```

Comment:
@Rohan I made this change because our popup code relies on this css class to handle submit buttons correctly.